### PR TITLE
Remove Presentation from home_sidebar.yml

### PR DESCRIPTION
**Pull request by SteveKola**
**Reference Issues:**
#134

**What does this implement/fix?**
Remove `Presentation` as it is an unnecessary duplication of `Introduction`. 

Signed-off-by: Steven Kolawole <kolawolesteven99@gmail.com>

### CODE REVIEW

**Comment by SteveKola** [comment]
@vchrombie, could you possibly help explain what I did wrong?

I filled in the DCO, I think.  I might have done something wrong though, so I can't be too sure since it's my first time.

Thank you.

**Comment by vchrombie** [comment]
Hi @SteveKola, the CHAOSS Charter requires that contributions are accompanied by a DCO sign-off. You can read more about it from here, [CONTRIBUTING.md](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md#dco-and-sign-off-for-contributions).

I'm not really sure about the exact reason for this error and I have seen it a couple of times, at other places, too. Well, we have a way to fix it but it requires a command line.
```
$ git clone https://github.com/SteveKola/grimoirelab-tutorial/
$ git checkout patch-1
$ git commit --amend --no-edit --signoff
$ git push --force-with-lease origin patch-1
```
Let me know if you need some help.

**Comment by SteveKola** [comment]
Thank you, @vchrombie.  That was super-helpful.

So I was able to guess what the error was. Apparently, DCO was expecting my GitHub-assigned email, not my regular email address that I entered. 

Thanks so much.

File: _data/sidebars/home_sidebar.yml [hunk 1]
```diff
@@ -9,10 +9,6 @@ entries:
     output: web
     folderitems:
 
-    - title: Presentation
-      url: /
-      output: web
-
     - title: Introduction
       url: /README.html
```
Comment (vchrombie) [inline _data/sidebars/home_sidebar.yml:13]: Can you change this field to `url: /` as README.html would also redirect to the same page?

**Review by vchrombie — CHANGES_REQUESTED** [review]
Thanks for the work, just had some small comments.

After changing the URL field, can you remove the first signoff from the commit message body?
```
 Remove Presentation from home_sidebar.yml

Signed-off-by: Steven Kolawole <kolawolesteven99@gmail.com>
Signed-off-by: stevekola <kolawolesteven99@gmail.com>
```

You can add the file and amend it to the same commit.
```
$ git add _data/sidebars/home_sidebar.yml 
$ git commit --amend
$ git push -f origin patch-1
```

Please feel free to add a small commit message body explaining in brief about the change and why is it needed. Let me know if you need any help. :smile:

**Comment by vchrombie** [comment]
Thanks for the work @SteveKola, we are close to making this a perfect PR.
It would be great if you can squash these two commits into one so that we can go for a final review and then merge it.

```
$ git clone https://github.com/SteveKola/grimoirelab-tutorial/
$ git checkout patch-1
$ git reset --soft HEAD~2 && git commit -s
$ git push -f origin patch-1
```

It would be great if you can write a good commit message and body describing the change. You can check this awesome blog post, https://chris.beams.io/posts/git-commit/.

**Comment by SteveKola** [comment]
Thank you for the blog post. It's awesome, truly.

I'd appreciate some feedback on the commit message and how it could be improved further.

**Comment by vchrombie** [comment]
Hi @SteveKola, the PR and commit message looks good but the commit has some additional changes that are pulled from the previous commits in the repo. I think you have added more than 2 previous commits but that's fine, we can correct that.

You need to reset that commit, remove the unnecessary file and commit again. I can help with the commands.
```
$ git reset --soft HEAD~1
$ git restore --staged cases-chaoss internals sirmordred
$ git restore cases-chaoss internals sirmordred
$ git commit -s
$ git push -f origin patch-1
```
Let me know if you need more help. :slightly_smiling_face:

**Comment by SteveKola** [comment]
> Hi @SteveKola, the PR and commit message looks good but the commit has some additional changes that are pulled from the previous commits in the repo. I think you have added more than 2 previous commits but that's fine, we can correct that.
> 
> You need to reset that commit, remove the unnecessary file and commit again. I can help with the commands.
> 
> ```
> $ git reset --soft HEAD~1
> $ git restore --staged cases-chaoss internals sirmordred
> $ git restore cases-chaoss internals sirmordred
> $ git commit -s
> $ git push -f origin patch-1
> ```
> 
> Let me know if you need more help.

Done. 
Thank you for the help, @vchrombie. I honestly didn't understand what I did wrong. 

And I must say, your in-depth understanding of Git commands is extremely impressive. Hopefully, I'd reach this level of proficiency one day. 

One other thing, the PR said "closed", did you close it from your end?

**Comment by SteveKola** [comment]
Seems all the changes were reverted. Do I need to make the changes again, and then push?

**Comment by vchrombie** [comment]
Hi @SteveKola 

> Seems all the changes were reverted. Do I need to make the changes again, and then push?

Yeah :slightly_frowning_face: 
Can you make the changes again in the same branch. The PR will be automatically opened after you push the commit.

Let me know if you want to connect in the IRC for some time to complete this.

**Comment by SteveKola** [comment]
Hi @vchrombie,

I made the changes, pushed, but this PR didn't open.
So I went to make the PR from my GitHub page, and it created a new PR.
#136 

I'd love to connect in the IRC so this could be wrapped up.

**Comment by vchrombie** [comment]
> So I went to make the PR from my GitHub page, and it created a new PR.

Now that it is created and this is closed, it's fine. We can go with that.

**Comment by SteveKola** [comment]
Thank you for your patience.

**Pull request closed without merging**